README.md

```diff
@@ -205,6 +249,78 @@ and you should see something along the lines of:
 
 ![tada](https://cdn.discordapp.com/attachments/616400094408736779/993705619498467369/ld-iZXysVXqwhM8.png)
 
+# enabling https traffic
+## using cloudflare (flex)
+first of all you need a cloudflare account, to create one go to https://dash.cloudflare.com/sign-up, enter your email and password and click on `create account`
+
+![Step 1](.github/images/ssl_cf_1.png)
+
+now you have to enter your domain. this has to be your actual domain (e.g. `banchopy.com` or `banchopy.net`) and mustn't include any domain-specific hostnames (e.g. `www.banchopy.com` or similar)
+
+![Step 2](.github/images/ssl_cf_2.png)
+
+then you have to choose your plan, for us it should be enough with the `Free plan`, you can also upgrade later if you need it
+
+![Step 3](.github/images/ssl_cf_3.png)
+
+now you'll have to copy the nameservers required by Cloudflare to your domain registrar. once you've done this, click on `check nameservers`
+
+![Step 4](.github/images/ssl_cf_4.png)
+
+once you have finished the above you'll have to add some dns records (A records) so that the necessary domains are being pointed to the ip where bancho.py is running
+
+<table>
+    <tr>
+        <th>
+        <ul>
+            <li>a.yourdomain.com</li>
+            <li>api.yourdomain.com</li>
+            <li>assets.yourdomain.com</li>
+            <li>c1.yourdomain.com</li>
+            <li>c2.yourdomain.com</li>
+            <li>c3.yourdomain.com</li>
+            <li>c4.yourdomain.com</li>
+            <li>c5.yourdomain.com</li>
+            <li>c6.yourdomain.com</li>
+            <li>ce.yourdomain.com</li>
+            <li>cho.yourdomain.com</li>
+            <li>c.yourdomain.com</li>
+            <li>yourdomain.com</li>
+            <li>i.yourdomain.com</li>
+            <li>map.yourdomain.com</li>
+            <li>osu.yourdomain.com</li>
+            <li>s.yourdomain.com</li>
+            <li>web.yourdomain.com</li>
+        </ul>
+        <th>
+            <img src=".github/images/ssl_cf_5.png" alt="Step 5">
+        </th>
+    </tr>
+</table>
+
+then go to SSL/TTL > overwiew and activate SSL flex
+
+![Step 6](.github/images/ssl_cf_6.png)
+
+it's also recommended to activate the option to always use HTTPS, you can enable it in SSL/TTL > edge certificates
+
+![Step 7](.github/images/ssl_cf_7.png)
+## using an own ssl certificate
+
+```sh
+# you'll need to change:
+# - YOUR_EMAIL_ADDRESS
+# - YOUR_DOMAIN
+
+# generate an ssl certificate for your domain
+sudo certbot certonly \
```

Comment:
honestly i would much rather recommend full (strict) cloudflare setup with an origin certificate over generating your own

Reply:
i assume you didn't see this before the latest commit @7mochi
also http to https could be done via nginx